app/services/api/v0_1/hesa_mapper/attributes.rb

```diff
@@ -194,19 +194,19 @@ def funding_entity_id
           ::Hesa::CodeSets::BursaryLevels::MAPPING[params[:funding_method]]
         end
 
-        def school_attributes
+        def lead_partner_and_employing_school_attributes
           attrs = {}
 
-          return attrs if params[:lead_school_urn].blank?
+          return attrs if params[:lead_partner_urn].blank?
 
-          if NOT_APPLICABLE_SCHOOL_URNS.include?(params[:lead_school_urn])
-            attrs.merge!(lead_school_not_applicable: true)
+          if NOT_APPLICABLE_SCHOOL_URNS.include?(params[:lead_partner_urn])
```

Comment:
Small thing, but should this constant now be `NOT_APPLICABLE_LEAD_PARTNER_URNS` or something like that?